bitnami/postgresql/Chart.yaml

```diff
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 12.8.2
+version: 12.8.3
```

Comment:
Version `12.8.3` has been already released but Github UI does not display it, could you please bump an additional patch version? (12.8.4)